images/build/debian-iptables/distroless/iptables-wrapper

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
This should be updated to be based on the latest version from https://github.com/kubernetes-sigs/iptables-wrappers/. (Or rather than "based on", you could actually use the `iptables-wrapper-installer.sh` from there to generate the wrapper at build time.)

The rules in this version of the script won't work reliably once [KEP-3178](https://github.com/kubernetes/enhancements/issues/3178) starts simplifying the kubelet iptables rules. (Probably 1.27?)